common/web/gesture-recognizer/src/mouseEventEngine.ts

```diff
@@ -79,7 +78,13 @@ namespace com.keyman.osk {
       }
 
       this.preventPropagation(event);
-      this.onInputStart(InputEventCoordinate.fromEvent(event));
+      const coord = InputEventCoordinate.fromEvent(event);
+
+      if(!ZoneBoundaryChecker.inputStartOutOfBoundsCheck(coord, this.config)) {
+        this.disabledSafeBounds = ZoneBoundaryChecker.inputStartSafeBoundProximityCheck(coord, this.config);
+      }
```

Comment:
What is this doing? It's not obvious why you are doing this -- please add a code comment!

Reply:
This comment is a secondary target of this comment: https://github.com/keymanapp/keyman/pull/6843#discussion_r919571361

This addresses probably the most niche aspect of the OSK's touchpoint management logic, so I'm not exactly surprised by the request.  Re #6842's visual:  "If the touchpoint is within the larger set of dashed lines but not the smaller set, remember which side we started on."  And _that_ has a second half.  Not exactly the simplest thing to document using only words; this is part of why I spent as much time as I did on the host page's visual design.